cpp/CMakePresets.json

```diff
@@ -44,7 +64,51 @@
         "CMAKE_BUILD_TYPE": "Debug"
       }
     },
-
+    {
+      "name": "windows-cl-conda-release",
```

Comment:
I believe these should not be included in this PR given critical dependencies are not available via conda on Windows